### PR TITLE
Added documentation for property tag usage in KDocs

### LINKED ISSUE #302: Suggest using @property tag in KDoc instead of putting comment before property in constructor
```kotlin
class CliArgument @JsonCreator internal constructor(
        // short argument representation like -h
        @param:JsonProperty("shortName") private val shortName: String
)
```
should be
```kotlin
/**
 * @property shortName short argument representation like -h
 */
class CliArgument @JsonCreator internal constructor(
        @param:JsonProperty("shortName") private val shortName: String
)
```

### PR DESCRIPTION
**Pull request by petertrr**
### What's done:
* Updated documentation
* Updated build.gradle.kts for gradle plugin for better loading in IDE

This pull request closes #302

### CODE REVIEW

File: info/guide/diktat-coding-convention.md [hunk 1]
```diff
@@ -2095,4 +2123,71 @@ interface I {
 object O: I {
     override fun foo() {}
 }
+```
+# <a name="c8"></a> 8. Things that will be moved to a main guide later
```
Comment (akuleshov7) [inline info/guide/diktat-coding-convention.md:2127]: this should not be there yet, bad idea to generate the version of a codestyle

File: info/guide/guide-chapter-2.md [hunk 2]
```diff
@@ -53,6 +53,32 @@ fun isEmptyList(list: List<String>) = list.size == 0
...
+4. Class properties declared in the primary constructor should be documented using `@property` tag in the class KDoc.
+
+Incorrect example:
+```kotlin
+/**
+ * Class description
+ */
+class Example(
+ /**
+  * property description
```
Comment (akuleshov7) [inline info/guide/guide-chapter-2.md:65]: what about simple comments?
```
/*
* comment
*/
```
or 

`//`

Comment (petertrr) [reply info/guide/guide-chapter-2.md:47]: Added description

File: info/guide/diktat-coding-convention.md (hunk 1 above)
Comment (petertrr) [reply info/guide/diktat-coding-convention.md]: OK, reverted

**Comment by petertrr** [comment]
In draft until changes in docmentation stabilize (#521 )

**Review by akuleshov7 — APPROVED** [review]
lgtm

**Pull request merged**